### ISSUE TITLE
Migration to a new CI

### ISSUE THREAD

**Issue by Vkt0r** [NoTests/RxFeedback.swift#24]
The current CI we have in this project is TravisCI, for some reason lately TravisCI had presented a lot of issues with the builds taking a lot of time to build 😢. CircleCI can be one the best choices out there and they have support for OSS projects.

I was talking with a Support Engineer for CircleCI and he told me that definitely, they support OSS projects in his Seed plan for OSX. I post what we need to add the project to CircleCI:

> What you would need to do is sign up for an account if you have not already. Once you have done that, add the projects to your account here: 
https://circleci.com/projects/gh/YOURUSERNAME

>Then if you could just provide me a link to your project I would be happy to enable the Seed plan for you.

>Your project link will look like this
https://circleci.com/gh/YOURUSERNAME/PROJECTNAME/

So pretty much after we had done that I can send it the link to he enable the Seed for the project. Please let me know guys what do you think about it. Of course, another options of CI are under the table if we want to discuss it.

**Comment by kzaher** [comment]
Hi @Vkt0r ,

do you have prior experience with CicleCi? I think I've added it to https://circleci.com/gh/NoTests/RxFeedback

Can you maybe help us setup configuration file?

**Comment by freak4pc** [comment]
I'd love to help. I've been doing it on all RxSwiftCommunity repos by now :)

**Comment by kzaher** [comment]
@freak4pc oh, ok. I've sent an owner invite.

**Comment by Vkt0r** [comment]
@kzaher @freak4pc Yes absolutely 👍, I have several private projects in Circle CI. Glad to help @freak4pc  what do you have in mind ?? I think it's best if we use Circle 2.0 directly, the pipelines are really great with it.

**Comment by Vkt0r** [comment]
@kzaher Let's start adding the project to the OSS macOS seed plan. Let me write it to the Support Engineer to add the project to the free macOS plan.

**Comment by freak4pc** [comment]
@Vkt0r Sounds great, we won't be able to move anywhere without that being added to the Organization unfortunately.

Regarding the solution itself, let's chat on RxSwift slack if you want - 

I think CircleCI 2.0 is overkill for macOS Builds in general (the YAML they use for workflows is so convoluted and of little use to this sort of project), I think just "converting" the existing travis.yml file to a CircleCI v1 file will work super-well here (just, split up the build phases into separate phases [instead of putting everything in a giant `eval`](https://github.com/NoTests/RxFeedback.swift/blob/master/.travis.yml#L19))

**Comment by Vkt0r** [comment]
@freak4pc @kzaher It's done the engineer activated the seed plan for the Organization `NoTest` according to what she told me !!! We can start the CI party 🎉. Sure @freak4pc @kzaher we can chat in the Slack channel, let's create a thread. 

Regarding the CircleCI 2.0 YML file you're right that for this project or any other library maybe it doesn't fit well into the workflows of CI 👍. Let's chat more about it
